### DISCUSSION TITLE
Akira can be used as an interaction designer

### DISCUSSION THREAD

**Discussion by abienz** [akiraux/Akira#513] [Ideas]
While initially Akira is targetting feature parity with other vector art programs, I would like to see that it can also be used to create short UI animations that can be exported to video or gif/png files.

There aren't many good alternatives out there and one of the most popular tools is Adobe After Effects! Clearly something that is as overkill for Interaction Design as Photoshop/Illustrator is for Web Design.

**Comment by albfan** [comment]
I would love to see something like this on Akira

https://mockups.snwh.org/gnome/patterns/lists

But is not something for our first release iterations. Not sure if it would be a goal at all for Akira.

@abienz if you can share an example would be great to figure out the details.

**Comment by abienz** [comment]
Hi, take a look at this link for some Interaction Design examples: https://icons8.com/articles/interaction-design-basic-types-ui-animation/

These animations are all over sites like Behance and Dribbble, and the majority are made with Adobe After Effects.

To be clear, I'm requsting a tool that can create these micro interaction examples and export as video, GIF, PNG sequence. Not to export as code for developers.

**Comment by julientaq** [comment]
I second what @abienz says. One of the most important thing about UX is that it's all about interaction, and some needs animation.
Being able to show animation inside the tool would be pretty amazing and a great feature. 

Since CSS and SVG are pretty good to show animations and interaction, maybe there is something down that road.

@abienz a bit unrelated, but you may be interested by this: https://github.com/mrachinskiy/commotion

**Comment by julientaq** [comment]
This is the kind of animation i'd like to send to developer. (i code those using css, but it would be great for designer to be able to share those assets without having to code anything)

https://codepen.io/julientaq/pen/NLyqJY

**Comment by Alecaddd** [comment]
I'm 100% on board with this!
As @albfan correctly said, we won't be able to ship this in the first release, not sure even if it will happen in the second release :sweat_smile: but it's definitely something we want to have in Akira.
Sketch, Figma, and InvisionStudio all have this, why can't we? :wink:
